Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM --platform=${BUILDPLATFORM:-linux/amd64} brew.registry.redhat.io/rh-osbs/openshift-golang-builder:v1.24 AS builder
+FROM --platform=${BUILDPLATFORM:-linux/amd64} brew.registry.redhat.io/rh-osbs/openshift-golang-builder:rhel_9_golang_1.24_test AS builder
```

Comment:
`rhel_9_golang_1.24_test` is a red flag. Is this a valid image to use? Why does the name contain `test`?

Reply:
So are we just keeping this and going to release with this image?